config/LAGraph.h.in

```diff
@@ -102,9 +102,15 @@
     #error "If using SuiteSparse::GraphBLAS, version 9.0.0 or later is required"
     #endif
     #define LAGRAPH_SUITESPARSE 1
+    #if GxB_IMPLEMENTATION >= GxB_VERSION (10,0,0)
+        #define USING_GRAPHBLAS_V10 1
```

Comment:
This #define of USING_GRAPHBLAS_V10 should not be exposed to the end user since it does not have the right name for that.  This #define should be moved to LG_internal.h, or if it appears in LAGraph.h it must be renamed to LAGRAPH_SUITESPARSE_V10 perhaps.  Better to just move it to LG_internal.h for now.